File: Programmers/Lv0/OX퀴즈.py
def solution(quizs):
    answer = []
    for quiz in quizs:
        expr = quiz.split() #expr의 크기는 항상 5. 1.피연산자1 2.연산자 3.피연산자2 4.등호 5.결과값
        num1 = int(expr[0])
        num2 = int(expr[2])
        if expr[1] == "+":
            num1 += num2
        else:
            num1 -= num2
        answer.append("O") if num1 == int(expr[4]) else answer.append("X")
    return answer

File: Programmers/Lv0/OX퀴즈.cpp
#include <string>
#include <vector>
#include <sstream>
using namespace std;

vector<string> solution(vector<string> quizs) {
    vector<string> answer;
    for (string quiz : quizs) {
        istringstream iss(quiz);
        string num1, num2, op, equl, ret;

        iss >> num1 >> op >> num2 >> equl >> ret;
        int n1 = stoi(num1), n2 = stoi(num2);
        if (op == "+") n1 += n2;
        else if (op == "-") n1 -= n2;
        n1 == stoi(ret) ? answer.push_back("O") : answer.push_back("X");
    }
    return answer;
}